mappings/net/minecraft/client/render/entity/feature/ArmorBipedFeatureRenderer.mapping

```diff
@@ -1 +1,6 @@
 CLASS net/minecraft/class_987 net/minecraft/client/render/entity/feature/ArmorBipedFeatureRenderer
+	METHOD method_4170 (Lnet/minecraft/class_572;Lnet/minecraft/class_1304;)V
+		ARG 1 bipedEntityModel
+		ARG 2 equipmentSlot
+	METHOD method_4190 (Lnet/minecraft/class_572;)V
```

Comment:
Is this a lambda parameter mapping entry?

Reply:
No but these methods are overriding methods